test/integration/helper.go

```diff
@@ -19,8 +19,8 @@ import (
 )
 
 type BlockExplorerTestSuite struct {
-	c  connectionmanager.ConnectionManager
-	be betest.BlockExplorerTestSetUp
+	C  connectionmanager.ConnectionManager
```

Comment:
actually, C is not readble from test. In the future will be better to use Mngr or ConMngr. 
So, if we call `a.ConMngr.Start()` that is more clear